simapp/state.go

```diff
@@ -14,6 +14,7 @@ import (
 	"github.com/cosmos/cosmos-sdk/codec"
 	simapparams "github.com/cosmos/cosmos-sdk/simapp/params"
 	"github.com/cosmos/cosmos-sdk/types/module"
+	simtypes "github.com/cosmos/cosmos-sdk/types/simulation"
 	"github.com/cosmos/cosmos-sdk/x/auth"
 	"github.com/cosmos/cosmos-sdk/x/simulation"
```

Comment:
Wait, this should be gone

Reply:
I was referring to line 19

```
"github.com/cosmos/cosmos-sdk/x/simulation"
```

Reply:
And 18

```
"github.com/cosmos/cosmos-sdk/x/auth"
```

Reply:
Ah ok @alessio , I thought it was only needed for the module package. So it is a dependency that not top package should depend on no? Modules should be something that is not dependant by any top module. Got it! But we can split in branches, this one only for simulation.

Reply:
OK, you're right. As things stand now, you've successfully removed `types` -> `x` dependencies. And this is already great progress :+1: